pylint/checkers/base/basic_checker.py

```diff
@@ -453,11 +453,14 @@ def visit_expr(self, node: nodes.Expr) -> None:
 
         # Warn W0133 for exceptions that are used as statements
         if isinstance(expr, nodes.Call):
-            inferred = utils.safe_infer(expr)
-            if isinstance(inferred, objects.ExceptionInstance):
-                self.add_message(
-                    "pointless-exception-statement", node=node, confidence=INFERENCE
-                )
+            name, _, _ = expr.as_string().partition("(")
```

Comment:
```suggestion
            name = expr.call.as_string()
```
That might work too.